Detectors/EMCAL/base/src/ClusterFactory.cxx

```diff
@@ -50,6 +58,8 @@ o2::emcal::AnalysisCluster ClusterFactory<InputType>::buildCluster(int clusterIn
   if (clusterIndex >= mClustersContainer.size()) {
     throw ClusterRangeException(clusterIndex, mClustersContainer.size());
   }
+  if (!mGeomPtr)
+    throw GeometryNotSetException();
```

Comment:
we need to use braces for if-statements (see full-CI).

Secondly, I think we prefer (previous discussion) the simpler variant of using LOG(fatal) for these kinds of problems unless the exception can actually be handled in a good way. It is also a lot less code.

Reply:
Yes, in many case they can, i.e. the task can (and should) be delegated to an outer component. Therefore I would prefer keeping the exception.

I will fix the braces.